### PR TITLE
TES-16825: Screenshots

### PR DESCRIPTION
**Pull request by barspielberg**
https://editor.swagger.io/?url=https://raw.githubusercontent.com/testimio/public-openapi/TES-16825-add-screenshots/api.yaml

### CODE REVIEW

**Comment by MoLow** [comment]
חשוב

File: api.yaml [hunk 1]
```diff
@@ -1189,6 +1189,20 @@ paths:
...
+                                  baseline: 
+                                    type: string
+                                  result: 
+                                    type: string
+                                  highlightedBaseline: 
+                                    type: string
+                                  highlightedResult: 
+                                    type: string
+                                  accessibleUntil: 
+                                    type: number
```
Comment (omril1) [inline api.yaml]: integer

**Review by liorama — APPROVED** [review]

File: api.yaml [hunk 2]
```diff
@@ -1411,6 +1425,120 @@ paths:
           '429':
             $ref: '#/components/responses/TooManyRequests'
     
+    /images/screenshots/{projectId}/{testId}/{stepId}/baseline:
```
Comment (RoySegall) [inline api.yaml]: If we're providing an endpoint for the baseline and an endpoint for the result are we deprecating the `screenshot` property from the steps on the execution?

**Review by RoySegall — APPROVED** [review]

File: api.yaml (hunk 2 above)
Comment (barspielberg) [reply api.yaml]: we plan to do that soon🤞

**Pull request merged**